sample/src/main/java/org/aerogear/graphqlandroid/ConflictPojo.kt

```diff
@@ -0,0 +1,28 @@
+package org.aerogear.graphqlandroid
+
+class ConflictPojo(
+    val errors: ArrayList<Errors>
+)
+
+class Errors(
+    val extensions : Extensions
+)
+
+class Extensions (
+    val exception: ExceptionClass
+)
+class ExceptionClass(
+    val conflictInfo : ConflictData
+)
+
+class ConflictData(
+    val serverState : ServerState
+)
+
+class ServerState(
```

Comment:
Do people need to define pojo in their app fr each type? Why this way? any reason we need this.

Reply:
Not in their app, we would be having it in the library.